io.github.mrmyhuang.cbetar2.yml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.mrmyhuang.cbetar2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=host
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: cbetar2
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/cbetar2/flatpak-node/cache
+        npm_config_cache: /run/build/cbetar2/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: 'true'
+    sources:
+      - type: archive
+        url: https://github.com/MrMYHuang/cbetar2/archive/4.26.6.tar.gz
+        sha256: e6ddfccbd1402fb5b41e8abd7da82324db8f9606415774669a36bcfd5570c23f
+
+      #- type: dir
+      #  path: ..
+      #  dest: main
+      - generated-sources.json
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/main/cbetar2 "$@"
+
+      - type: file
+        path: io.github.mrmyhuang.cbetar2.metainfo.xml
+
+      - type: file
+        path: io.github.mrmyhuang.cbetar2.desktop
+
+      - type: file
+        path: io.github.mrmyhuang.cbetar2.png
```

Comment:
the repository has this file so you shouldn't need to add it to the repository

Reply:
fix